cli_tools/gce_vm_image_import/importer/bootable_disk_processor.go

```diff
@@ -37,8 +37,8 @@ type bootableDiskProcessor struct {
 	OS              string
 }
 
-func (b bootableDiskProcessor) process(ctx context.Context) (err error) {
-	err = b.workflow.RunWithModifiers(ctx, b.preValidateFunc(), b.postValidateFunc())
+func (b *bootableDiskProcessor) process(ctx context.Context) (err error) {
```

Comment:
Because the receiver is copied on each call. What's the motivation not to use a pointer?

Reply:
> Because the receiver is copied on each call.

Were you seeing pressure on the GC? The general guidance is to reserve pointers for mutability, which serves for self-documentation (danger! things change!), and safety, to ensure thing's *don't* change when when unexpected. 

go/gocomments#pass-values:  "Don't pass pointers as function arguments just to save a few bytes."

Reply:
That link is about func arguments. Receiver is a special type of an argument where function "belongs" to a struct and there's a high chance it will modify it. Using pointers consistently for receivers will reduce hard to trace errors where, a function that once only used the receiver, needs to modify something, but the modifications don't stick because they were made to a copy. I actually had this situation in this code where receivers are a mix of pointers and values. I find that mix very messy.

Reply:
https://tour.golang.org/methods/8

https://golang.org/doc/faq#methods_on_values_or_pointers

_If some of the methods of the type must have pointer receivers, the rest should too, so the method set is consistent regardless of how the type is used._

Reply:
All of the receivers for bootableDiskProcessor currently *are* values, though -- what were you wanting to modify?

Reply:
I think for receivers we should always use pointers for consistency. I don't see a strong reason not to. More likely than not there will be a need to modify the receiver since functions belong to it. When that happens, all receivers need to be changed. It can also lead to hard to debug bugs where changes to a receiver don't stick because it's a value. 

The only benefit for using value receivers is to be explicit changes will not be made, which is dubious for receivers IMO. Any other benefits?

Reply:
Mutable objects make even *harder*-to-debug situations, and there's a wealth of research and work to avoid them. What're you wanting to mutate here?

- https://github.com/google/guava/wiki/ImmutableCollectionsExplained
- https://github.com/google/auto/blob/master/value/userguide/index.md
- https://research.swtch.com/go2017#immutability

Reply:
I already explained it's for consistency across our code base, never said I want to mutate anything at this right moment.

Reply:
I'm very interested to learn about your view on exclusive-pointer receivers, and we should continue the discussion on its own. Given that this current PR doesn't need that functionality, though, it's probably best to forego those refactoring changes for now.

Reply:
So, I'm still seeing receiver changes in this version. For this PR, let's focus on the current issue, and then save refactorings and other items for future PRs.

Reply:
Removed pointer receivers on processors.